.gitignore

```diff
@@ -2,4 +2,4 @@ node_modules/
 .DS_Store/
 .DS_Store
 log.txt
-log.txt
+.DS_Store
```

Comment:
Could you please explain why we have a couple of `DS_Store` instances in this file.

Reply:
Not really. I guess i commited it once and tried to ignore afterwards